tests/lib/arista_test.py

```diff
@@ -171,7 +171,8 @@
         'version-2-multicast-listener-report',
     },
     'option': {'established',
-               'tcp-established'}
+               'tcp-established',
```

Comment:
Do these other platforms actually support this?

Reply:
I really don't know, but the test fails because it has Cisco dependencies.